src/adapters/models/deberta/mixin_deberta.py

```diff
@@ -14,3 +15,8 @@ def init_adapters(self, model_config, adapters_config):
             self.location_key + "_prefix" if self.location_key else None, model_config, adapters_config
         )
         patch_forward(self)
+
+
+class DebertaModelAdaptersMixin(BertModelAdaptersMixin):
+    # Same as BERT, except that Deberta does not support the "lora_delta_w_svd" combine_strategy
+    support_lora_delta_w_svd = False
```

Comment:
Why is DeBERTa not supporting lora_delta_w_svd?

Reply:
The hugging face transformers implementation of Deberta and GPT-2 differs from that of other models. Therefore, we also use `LoRAMergedLinear` in their mixins and not `LoRALinear`. But since this implementation differs significantly from LoRALinear, we currently do not support composition blocks of LoRA and (IA)³ for GPT-2 and Deberta (see [here in our docs](https://docs.adapterhub.ml/adapter_composition.html#composition-blocks-overview)). Our `average_adapter` method encounters the same problems as the composition blocks do, hence Deberta and GPT-2 do not support the `lora_delta_w_svd` combination strategy.